library/debug.c

```diff
@@ -246,7 +246,7 @@ void mbedtls_debug_print_mpi( const mbedtls_ssl_context *ssl, int level,
     else
     {
         int n;
-        for( n = ( bitlen - 1 ) / 8; n >= 0; n-- )
+        for( n = (int) ( bitlen - 1 ) / 8; n >= 0; n-- )
```

Comment:
I'm not saying this is incorrect, but wouldn't it be simpler to just declare `n` as a `size_t` on the line above? Any particular reason for sticking to `int` instead?

Reply:
It's a decreasing for loop, the termination condition is `n >= 0`.

Reply:
Duh. Sorry for the noise.